geode-for-redis/src/distributedTest/java/org/apache/geode/redis/internal/executor/string/StringsKillMultipleServersDUnitTest.java

```diff
@@ -49,10 +50,19 @@
   @BeforeClass
   public static void classSetup() {
     MemberVM locator = cluster.startLocatorVM(0);
-    cluster.startRedisVM(1, locator.getPort());
-    cluster.startRedisVM(2, locator.getPort());
-    cluster.startRedisVM(3, locator.getPort());
-    cluster.startRedisVM(4, locator.getPort());
+    int locatorPort = locator.getPort();
+    cluster.startRedisVM(1, x -> x
+        .withConnectionToLocator(locatorPort)
+        .withProperty(REDUNDANCY_ZONE, "A"));
+    cluster.startRedisVM(2, x -> x
+        .withConnectionToLocator(locatorPort)
+        .withProperty(REDUNDANCY_ZONE, "B"));
+    cluster.startRedisVM(3, x -> x
+        .withConnectionToLocator(locatorPort)
+        .withProperty(REDUNDANCY_ZONE, "A"));
+    cluster.startRedisVM(4, x -> x
+        .withConnectionToLocator(locatorPort)
+        .withProperty(REDUNDANCY_ZONE, "B"));
```

Comment:
My understanding of redundancy zones may be wrong, but I think this change doesn't necessarily guarantee that we lose all copies of a bucket when crashing server3 and server4. If every secondary copy for server3 is on server2 and every secondary copy for server4 is on server1 (which is what will happen if we define these redundancy zones) then we can crash server3 and server4 but still have at least one copy of every bucket survive. I don't know how easy it would be to do, but we could be sure of crashing the "right" servers by finding which servers host the primary and the secondary for an arbitrary bucket and crashing both.

Reply:
So in that scenario the test would still pass but wouldn't be testing the right thing. However given that the likelihood of that happening is so incredibly small (pretty sure its around 1:2^64) I don't think it's worth over-complicating the test.